File: CS/MethodToCommandExample/ViewModel/MainViewModel.cs
using System;
using System.Linq;
using DevExpress.Mvvm.POCO;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using MethodToCommandExample.Common;

namespace MethodToCommandExample.ViewModel {
    public class MainViewModel {
        public MainViewModel() {
            Users = new ObservableCollection<User>() {
                User.Create(0, "Jack"),
                User.Create(1, "Ron"),
                User.Create(2, "John"),
                User.Create(3, "Antoni"),
                User.Create(4, "Paul"),
            };
        }

        public static MainViewModel Create() {
            return ViewModelSource.Create(() => new MainViewModel());
        }

        public virtual ObservableCollection<User> Users { get; set; }
    }
}

File: VB/MethodToCommandExample/ViewModel/MainViewModel.vb
Imports DevExpress.Mvvm.POCO
Imports System.Collections.ObjectModel
Imports MethodToCommandExample.Common

Namespace MethodToCommandExample.ViewModel

    Public Class MainViewModel

        Public Sub New()
            Users = New ObservableCollection(Of User)() From {User.Create(0, "Jack"), User.Create(1, "Ron"), User.Create(2, "John"), User.Create(3, "Antoni"), User.Create(4, "Paul")}
        End Sub

        Public Shared Function Create() As MainViewModel
            Return ViewModelSource.Create(Function() New MainViewModel())
        End Function

        Public Overridable Property Users As ObservableCollection(Of User)
    End Class
End Namespace
